pkg/controller/kogitodataindex/resource/testdata/onboarding-dockerimage-json

```diff
@@ -36,7 +36,7 @@
             "-c",
             "while [ ! -f \"/tmp/s2i-upload-done426790037\" ]; do sleep 0.5; done; /usr/local/s2i/assemble-runtime; exit $?"
         ],
-        "Image": "quay.io/kiegroup/kogito-quarkus-jvm-ubi8:0.5.0",
+        "Image": "quay.io/kiegroup/kogito-quarkus-jvm-ubi8:0.6.0",
```

Comment:
maybe consider to use 0.6.1, it will be ready by Monday.

Reply:
I rebased from master and did not change any version from it. It should be directly update or should I do it here ? (but who knows when it will be merged ...)